python/tests/test_eval.py

```diff
@@ -188,6 +190,79 @@ async def test_run_test_suite_with_inputs(data: st.DataObject):
             assert False, f"expected Ok, got Err({e})"
 
 
+@hypothesis.given(st.data())
+@pytest.mark.asyncio
+async def test_run_test_suite_with_inputs_general_params(data: st.DataObject):
+    """In test_run_test_suite_outputs_only, we test the user-facing function (e2e)
+    In this case that's harder because run_test_suite_with_inputs takes
```

Comment:
Couldn't you mock fs load of the path to return mock json?

Reply:
Yeah I guess I could. Adding here